static/style.css

```diff
@@ -0,0 +1,145 @@
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.navbar {
+  width: 100%;
+  overflow: auto;
+  justify-content: space-between;
+}
+
+.logo {
+  background-image: url("/Users/mrblue/Documents/projects/mb-portfolio/static/logo.png");
```

Comment:
- [ ] I notice that there is a bit of vertical and horizontal space around the background image, which isn't in the design, Kindly switch the background image from the logo to the header to attain a pixel perfect design as such.

```

.header {
  background-image: url("./logo.png");
  background-repeat: no-repeat;
  display: flex;
  align-items: center;
  padding: 10px 24px;
  justify-content: space-between;
  background-color: white;
  position: fixed;
  width: 100%;
}
```

![mobile](https://user-images.githubusercontent.com/41163571/236009037-b4a80fc0-95d4-42aa-b32c-d885000252a4.gif)

Reply:
You should be able to figure out the alignment of the bio content on your own I guess 😉

Have a nice one.